message_definitions/v1.0/ras_a.xml

```diff
@@ -10,7 +10,140 @@
+        <param index="2" label="Time limit" units="s" minValue="0">Time limit to execute the exploration. Reaching this time triggers the behavior defined in param 4. Set to 0 when there is no time limit.</param>
+        <param index="3" label="Behavior after stopping" minValue="0" maxValue="2" increment="1">The behavior after stopping the exploration task. 0: Hold, 1: Land, 2: Return to position.</param>
+        <param index="4" label="Ingress Portal ID" minValue="0" maxValue="4294967295" increment="1">ID of the ingress portal. If the ID is unknown, not deterministic, or when already in the area to explore after going through a portal, set it to UINT32_MAX and use params 5, 6 and 7 to define the local or global position of the portal. Note that an egress portal is not defined by this command but can be consequently set either by the vehicle autonomy engine or by an operator/external controller using MAV_CMD_SET_EGRESS_PORTAL.</param>
+        <param index="5" label="Position X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the portal position. NaN or INT32_MAX if unknown (or if using param 4 to set the portal).</param>
+        <param index="6" label="Position Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the portal position. NaN or INT32_MAX if unknown (or if using param 1 to set the portal).</param>
+        <param index="7" label="Position Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the portal position. NaN when unknown or when using param 4 to set the portal ID.</param>
+      </entry>
+      <entry value="40511" name="MAV_CMD_STOP_EXPLORATION" hasLocation="false" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Stop an exploration task. The behavior after stopping is defined by a parameter
+           Return to position should take the vehicle to the defined exit portal first, and then to the return post-exploration point (accessible through EXPLORATION_RETURN_POSITION).
+           If the passed exploration ID does not exist or is not listed as a valid exploration ID in the vehicle autonomy engine, then this command should be rejected.
+        </description>
+        <param index="1" label="Exploration task ID" minValue="0" maxValue="4294967295" increment="1">Sets the ID of the (current or queued) exploration task to stop. Set to UINT32_MAX if one wants to stop the current exploration task.</param>
+        <param index="2" label="Behavior after stopping" minValue="0" maxValue="2" increment="1">The behavior after stopping the exploration task. 0: Hold, 1: Land, 2: Return to position.</param>
+        <param index="3" reserved="true" default="NaN"/>
+        <param index="4" reserved="true" default="NaN"/>
+        <param index="5" reserved="true" default="NaN"/>
+        <param index="6" reserved="true" default="NaN"/>
```

Comment:
```suggestion
        <param index="5" reserved="true" default="0"/>
        <param index="6" reserved="true" default="0"/>
```
YOu can't default an int to a Nan

Reply:
If this is sent as a COMMAND_LONG, then I can. But let's make it flexible to be used with both INT and LONG, so let's default to 0.